### DISCUSSION TITLE
Include userLocation with fitToElements

### DISCUSSION THREAD

**Discussion by jreason** [react-native-maps/react-native-maps#4587] [Feature requests]
Hi,

I have 1 marker and when i run fitToElements it zooms to the marker which is great.
However this does not seem to take into account the fact that i am using:
showsUserLocation
followsUserLocation

I want to be able to see any markers along with the users current location all in the same view.

It would be great to have an extra option whether to take into account the users current location?

options: { edgePadding: EdgePadding, animated: Boolean, includeUsersLocation: Boolean }

**Comment by lucksp** [comment]
can you share more of your code?  What are the user's coordinates?

In theory, as long as you have `showUserLocation` enabled, the map should automatically fit all the contents.  I am trying to do the opposite, where I don't necessarily want the map to "fit" with the user location
